tests/S3Control/S3ControlEndpointMiddlewareTest.php

```diff
@@ -67,6 +67,21 @@ public function testDoesNothingWhenDualStackDisabledOnOperationLevel()
         $middleware($command, $this->getRequest($command));
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     */
```

Comment:
Should there be an expected message, to make sure we're failing on the right thing?

Reply:
Fixed in b456071cb501d1be6262151fb885118c4c7c82ab